plugins/cosmic_reach_model_editor/cosmic_reach_model_editor.js

```diff
@@ -0,0 +1,287 @@
+
+function sumArrays(a, b){
+    s = []
+    for(let i = 0; i < Math.min(a.length, b.length); i++){
+        s.push(a[i] + b[i])
+    }
+    return s
+}
+
+(() => {
+    let codec, format, export_action, import_action, dialog, properties
+    const id = "cosmic_reach_model_editor"
+    const name = "Cosmic Reach Model Editor"
+    const icon = "icon.png"
+    Plugin.register(id, {
+      title: name,
+      icon: "icon.png",
+      author: "Z. Hoeshin",
+      description: "Allows creating, editing, importing and exporting Cosmic Reach block models.",
+      tags: [],
+      version: "1.1.0",
+      creation_date: "2024-03-09",
+      onload() {
+        Codecs.java_block.load_filter.condition = (model) => {
+			return (model.parent || model.elements || model.textures) && (!model.cuboids);
+		}
```

Comment:
You should not directly overwrite the condition for the load filter, as its internal condition is subject to change whether inside Blockbench or an another plugin. You should store the original condition and make use of it inside the overwritten condition.

Define a reference in the plugin's scope.
```js
let originalJavaBlockLoadFilter;
```
```suggestion
        originalJavaBlockLoadFilter = Codecs.java_block.load_filter.condition;
        Codecs.java_block.load_filter.condition = (model) => {
		return originalJavaBlockLoadFilter(model)  && (!model.cuboids);
	}
```
And do not forget onunload to reset the condition..
```js
Codecs.java_block.load_filter.condition = originalJavaBlockLoadFilter;
```